src/main/java/model/Figure.java

```diff
@@ -0,0 +1,12 @@
+package model;
+
+import utility.AreaCalculator;
+import utility.PrintInfo;
+
+public abstract class Figure implements AreaCalculator, PrintInfo {
```

Comment:
Avoid using abstract classes to set behavior for classes. Consider whether `Figure` should be an abstract class or if an interface would be more appropriate for defining behaviors like `AreaCalculator` and `PrintInfo`.